File: Mavim-services/Admin/Mavim.Admin.Api.Import.Catalog/Model/DbConnectionInfo.cs
using System;
namespace Mavim.Admin.Api.Import.Catalog.Model
{
    public class DbConnectionInfo
    {
        public Guid Id { get; set; }
        public Guid TenantId { get; set; }
        public string DisplayName { get; set; }
        public string ConnectionString { get; set; }
        public string Schema { get; set; }
        public Guid? ApplicationTenantId { get; set; }
        public Guid? ApplicationId { get; set; }
        public string ApplicationSecretKey { get; set; }
        public bool IsInternalDatabase { get; set; }
    }
}

File: Mavim-apps/Mavim.admin/mavim-import-catalog/src/app/home/DbConnectionInfo.ts
export class DbConnectionInfo {
  public constructor(
    public displayName: string,
    public connectionString: string,
    public schema: string,
    public tenantId: string,
    public applicationTenantId: string,
    public applicationId: string,
    public applicationSecretKey: string,
    public isInternalDatabase: boolean
  ) {}
}
